macros/utils/cross_db_utils/generate_elementary_profile_args.sql

```diff
@@ -102,6 +102,19 @@
   {% do return(parameters) %}
 {% endmacro %}
 
+{% macro clickhouse__generate_elementary_profile_args(method, elementary_database, elementary_schema) %}
+  {% do return([
+    _parameter("type", target.type),
+    _parameter("host", target.host),
+    _parameter("port", target.port),
+    _parameter("user", target.user),
+    _parameter("password", "<PASSWORD>"),
+    _parameter("schema", elementary_schema),
+    _parameter("driver", "<http|native>"),
```

Comment:
is this available through the target (e.g `target.driver`)? if so i believe it would be better